code/go/0chain.net/smartcontract/dbs/event/block.go

```diff
@@ -15,7 +15,7 @@ type Block struct {
 	Hash                  string    `json:"hash"`
 	Version               string    `json:"version"`
 	CreationDate          int64     `json:"creation_date"`
-	Round                 int64     `json:"round"`
+	Round                 int64     `json:"round" gorm:"index:idx_bround,unique"`
```

Comment:
The block round is not unique if I remember correctly that we will save all proposed blocks to the event db.

Reply:
yeah, correct,